trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+# InnerSource Learning Path: Trusted Committer
+
+## Introduction
+
+- tbd
+
+## Introducing The Trusted Committer Role
+
+The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. It is both a challenging and a rewarding role to fulfill and it is
+instrumental for the success of any InnerSource community. 
+
+Generally speaking, the TC role is defined by its responsibilities, rather than
+by its privileges. On a very high level, a TC represents the interests of both
+their InnerSource community and the products the community is building. So as
+a TC, you'll have both tech oriented and community oriented responsibilities
+and we'll explore both of these dimensions in the following sections.
+
+Before we go into the details of what a TC actually does, let's spend some time
+contrasting the TC role to other roles in IS and why we think the name is both
+apt and important.
+
+- Contributor
```

Comment:
per my other comments, I think you could omit this part.  I would be interested in how a Contributor might be trained or up-leveled to become a TC.

Reply:
looks like you have it down below.  You might want to put that right after the TC section below this one.